instrumentation/opentelemetry-instrumentation-celery/src/opentelemetry/instrumentation/celery/__init__.py

```diff
@@ -215,6 +215,11 @@ def _trace_postrun(self, *args, **kwargs):
         self._record_histograms(task_id, labels)
         context_api.detach(token)
 
+        # if the process sending the task is not instrumented
+        # there's no incoming context and no token to detach
+        if token is not None:
```

Comment:
Sorry please ignore my review request;
One comment here; shouldn't this if statement precede line \#216?

Reply:
yup, I was reverting some formatter changes and reverted too much :grimacing: . Should be good now.